Align README install instructions and those in create tool

sarahec opened the pull request:
The README didn't quite match the setup instructions printed by the svutCreate (and the tool needs a $ before HOME). This patches both the tool and the README to be consistent.

Also patches .gitignore to allow Visual Studio Code as a

damofthemoon commented:
Thank you for this pull request. I am looking forward your next one or any review of SVUT 🤓

sarahec commented:
My next planned updates are:
1. Recognize tb_* and *_tb as well as *_unit_test (tb as in "test bench")
2. Add an option to find and run all tests in a folder
3. Add an option to watch for file or folder changes and rerun the tests
automatically

And looking for ways to make the create tool less sensitive to different
source formatting (i.e. a tiny lexer/parser).

I'm intending to use this for a free workshop next year.

On Mon, Jul 29, 2019, 23:22 Damien Pretet <notifications@github.com> wrote:

> Thank you for this pull request. I am looking forward your next one or any
> review of SVUT 🤓
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/damofthemoon/svut/pull/1?email_source=notifications&email_token=ACWBND2BUZWET3NRUSESSMDQB7M33A5CNFSM4IHV6OT2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD3C5ENI#issuecomment-516280885>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACWBND6NA6HJPGT3KWVYTV3QB7M33ANCNFSM4IHV6OTQ>
> .
>

damofthemoon commented:
* Point 1 is easy to do. I can do the update fast if you want.
* Point 2 should work if I am not wrong, maybe there is a bug here. The script svutRun.py parses the folder from which it’s launched and collect all files with unit_test.{v,sv} to run them.
* Point 3 looks really good, I am looking forward to see that alive!

For further updates, I would be glad you open an issue just to track easily contribution/feature and have a history. Thank you for your help! I really appreciate to see this personal project used by other people 😄

sarahec commented:
Point 2 is broken :( Do you want to fix that one?


On Tue, Jul 30, 2019 at 11:21 AM Damien Pretet <notifications@github.com>
wrote:

>
>    - Point 1 is easy to do. I can do the update fast if you want.
>    - Point 2 should work if I am not wrong, maybe there is a bug here.
>    The script svutRun.py parses the folder from which it’s launched and
>    collect all files with unit_test.{v,sv} to run them.
>    - Point 3 looks really good, I am looking forward to see that alive!
>
> For further updates, I would be glad you open an issue just to track
> easily contribution/feature and have a history. Thank you for your help! I
> really appreciate to see this personal project used by other people 😄
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/damofthemoon/svut/pull/1?email_source=notifications&email_token=ACWBND3IE4AQJTOA3PPOUHTQCCBA5A5CNFSM4IHV6OT2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD3E3GDQ#issuecomment-516535054>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACWBND4WDOZ76MMZJVMJNRDQCCBA5ANCNFSM4IHV6OTQ>
> .
>


-- 
Sarah Clark | Program Manager, DevRel Infrastructure | sarahclark@google.com
 | M: +1.408.605.2653

sarahec commented:
And you might as well include tb_ and _tb at the same time.

On Tue, Jul 30, 2019 at 11:45 AM Sarah Clark <sarahclark@google.com> wrote:

> Point 2 is broken :( Do you want to fix that one?
>
>
> On Tue, Jul 30, 2019 at 11:21 AM Damien Pretet <notifications@github.com>
> wrote:
>
>>
>>    - Point 1 is easy to do. I can do the update fast if you want.
>>    - Point 2 should work if I am not wrong, maybe there is a bug here.
>>    The script svutRun.py parses the folder from which it’s launched and
>>    collect all files with unit_test.{v,sv} to run them.
>>    - Point 3 looks really good, I am looking forward to see that alive!
>>
>> For further updates, I would be glad you open an issue just to track
>> easily contribution/feature and have a history. Thank you for your help! I
>> really appreciate to see this personal project used by other people 😄
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/damofthemoon/svut/pull/1?email_source=notifications&email_token=ACWBND3IE4AQJTOA3PPOUHTQCCBA5A5CNFSM4IHV6OT2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD3E3GDQ#issuecomment-516535054>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/ACWBND4WDOZ76MMZJVMJNRDQCCBA5ANCNFSM4IHV6OTQ>
>> .
>>
>
>
> --
> Sarah Clark | Program Manager, DevRel Infrastructure | s
> arahclark@google.com | M: +1.408.605.2653
>
>

-- 
Sarah Clark | Program Manager, DevRel Infrastructure | sarahclark@google.com
 | M: +1.408.605.2653

damofthemoon commented:
I added the new prefix/suffix you proposed and fixed the multiple test suites execution. Works on my side.

sarahec commented:
I can confirm your fix for multiple test suites works. Thank you for the
prefix/suffix changes as well!

On Tue, Jul 30, 2019 at 12:59 PM Damien Pretet <notifications@github.com>
wrote:

> I added the new prefix/suffix you proposed and fixed the multiple test
> suites execution. Works on my side.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/damofthemoon/svut/pull/1?email_source=notifications&email_token=ACWBND5A6WNSBVPQNHV3XBDQCCMQLA5CNFSM4IHV6OT2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD3FEL2I#issuecomment-516572649>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACWBND25I44SEGKHO6GSKHDQCCMQLANCNFSM4IHV6OTQ>
> .
>


-- 
Sarah Clark | Program Manager, DevRel Infrastructure | sarahclark@google.com
 | M: +1.408.605.2653
